share/locale/languages.xml

```diff
@@ -37,6 +37,7 @@
   <language code="it" name="Italiano" />
   <language code="ja" name="日本語" />
   <language code="ka" name="ქართული" />
+  <language code="kab" name="ⵜⴰⵇⴱⴰⵢⵍⵉⵜ" />

```

Comment:
On Windows, I believe MuseScore menu fonts is the default windows font. I don't see this string here on mac on github.